src/dynamics/initial_conditions.jl

```diff
@@ -315,18 +315,34 @@ function initialize!(
 )
     (; m, ω, K, c) = initial_conditions
     (; geometry) = model
+    Ω = model.planet.rotation
+    a = model.spectral_grid.radius
```

Comment:
you think we could rename the radius from `a` to `R`? That way it would be consistent with other parts of the model/documentation!

Reply:
I think it's now consistently called `R` for radius but `m` for order / the zonal wavenumber, could you check?